Cargo.toml

```diff
@@ -68,6 +68,10 @@ ipnet = { version = "2.7.0", features = ["serde"] }
 http-types = { version = "2.12.0", default-features = false }
 netns-rs = "0.1.0"
 textnonce = { version = "1.0.0" }
+boringssl = "0.0.5"
```

Comment:
need to use the existing `boring`, we already have a library for this

Reply:
sure, will check this.